bom/runtime/pom.xml

```diff
@@ -1665,6 +1665,11 @@
                 <artifactId>vertx-web-client</artifactId>
                 <version>${vertx.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.vertx</groupId>
+                <artifactId>vertx-web-graphql</artifactId>
+                <version>${vertx.version}</version>
```

Comment:
@cescoffier @vietj I wonder if we should include the Vert.x bom in Quarkus?

Reply:
+1 very likely any other artifact that is used would need to be added anyway, using the bom definitely solve for more use cases.

Reply:
Yes, that would avoid many additional dependencies. 

However, we don't have a `bom` named `bom`. It's called `dep-chain`. The dependency to add is:

```
<dependency>
    <groupId>io.vertx</groupId>
    <artifactId>vertx-stack-depchain</artifactId>
    <version>3.8.0</version> <!-- must be replaced with the Vert.x version -->
    <type>pom</type>
</dependency>
```

Reply:
@cristianonicolai I let you update your PR? We have a few boms already defined there: https://github.com/quarkusio/quarkus/blob/master/bom/runtime/pom.xml#L171

Reply:
sure, I can update my PR.